assets/src/edit-story/components/panels/stylePreset/test/panel.js

```diff
@@ -217,7 +214,7 @@ describe('Panels/StylePreset', () => {
           stylePresets: {
             textColors: [],
             fillColors: [],
-            textStyles: [STYLE_PRESET],
+            textStyles: [{ color: TEST_COLOR_2, ...STYLE_PRESET }],
```

Comment:
Just a color alone would not be part of `textStyles`, it would be part of `textColors`.

Reply:
But it's not a color alone - it's color and whatever's in `STYLE_PRESET` (assuming it's non-empty)?

Reply:
Whoops, saw incorrectly as if these were two separate array items.